config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -34,6 +34,46 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: pull-kubernetes-e2e-containerd-gce
+    always_run: false
+    optional: true
+    skip_branches:
+      - release-\d+\.\d+ # per-release image
+    annotations:
+      fork-per-release: "true"
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
+    spec:
+      containers:
+        - args:
+            - --root=/go/src
+            - --repo=k8s.io/kubernetes=$(PULL_REFS)
+            - --repo=k8s.io/release
+            - --upload=gs://kubernetes-jenkins/pr-logs
+            - --timeout=105
+            - --scenario=kubernetes_e2e
+            - --
+            - --build=bazel
+            - --cluster=
+            - --extract=local
+            - --env=KUBE_CONTAINER_RUNTIME=containerd
```

Comment:
You also need "LOG_DUMP_SYSTEMD_SERVICES=containerd containerd-installation" to dump containerd logs.

Reply:
Fixed.

Reply:
i threw in containerd-installation as well

Reply:
There won't be `containerd-installation` service anymore, because we've built containerd setup in the default kube-up scripts.

Reply:
Ack dropping that